### PR TITLE
Xpanse - Adding new VM playbook

### PR DESCRIPTION
**Pull request by andrew-paloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Adding a new playbook that allows for vuln lookups via Tenable.io for IPs/Domains

## Screenshots
N/A

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ExpanseV2/Playbooks/playbook-Expanse_VM_Enrich_README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| Vulnerability Exist | Identifies whether in the last 30 days a vulnerability exists in Tenable.io for the associated asset within the last 30 days | TenableIO.Vulnerabilities | Optional |
+
+## Playbook Outputs
+
+---
+
+| **Logic** | **Name** | **Description** | **Default Value** |
+| --- | --- | --- | --- |
+| Y - Finds Vulnerability | Prints vulnerability summary | Shows user the related information of the identified vulnerability for the asset | N/A |
+| N - Does not find Vulnerability | Manual VM scan | Initiates a task for the user to kick of a manual VM scan of the identified asset | N/A |
```
Comment (dantavori) [inline Packs/ExpanseV2/Playbooks/playbook-Expanse_VM_Enrich_README.md:37]: These are not actual playbook inputs and outputs, right? If so, let's describe this content at the description above. 
```suggestion
## Playbook Inputs
---
There are no inputs for this playbook.

## Playbook Outputs
---
There are no outputs for this playbook.
```

**Review by dantavori — CHANGES_REQUESTED** [review]
looks good to me, see my note below and please re-generate the release notes to also include the update in the integration (otherwise it won't pass the validation step in the build).
Let's talk on Slack to schedule a demo.

@EliorKedar FYI

**Review by EliorKedar — APPROVED** [review]

**Comment by dantavori** [comment]
@andrew-paloalto since version 1.3.0 of this pack is already on master you have a conflict, and you need to:
1. Remove the current release notes file from your branch (copy the content for later)
2. Pull from origin/master
3. run `demisto-sdk update-release-notes` again - it will generate the `1_4_0.md` file and will bump the pack version in the metadata to 1.4.0
4. paste the content of your original RN file.

We can merge the PR when this is fixed.

**Comment by andrew-expanse** [comment]
Hey @dantavori, the merge from master looks a bit ugly, but the version for ExpanseV2 has been bumped to 1.4.0 and I've added new release notes and bumped the UA header in the integration.

**Review by dantavori — APPROVED** [review]

**Pull request merged**